P1.txt

```diff
@@ -0,0 +1,12 @@
+val a: Integer
+read(a)
+
+if (a < 2) {
+	write(“false”)
+} else {
+	for (2 to a/2)

```

Comment:
By the syntax definition, for contains a statement, which in turn can be a compound statement, in tern being of the form "{" mixedStatement "}, so the block statement of the for should be surrounded by {}.